packages/components/LIMITATIONS.md

```diff
@@ -169,3 +169,81 @@ Try to match on the next `<syn-nav-item>` with the closest selector. This will a
   });
 </script>
 ```
+
+---
+
+## Wrong rendering order with container-type set on parent for `<syn-select>`, `<syn-combobox`, `<syn-dropdown>`, `<syn-tooltip>`
+
+### Meta Information
+
+- Framework version: ALL
+- Synergy version: ALL
+- Browsers: safari, older chrome and older firefox versions
+- Issues: [#612](https://github.com/synergy-design-system/synergy-design-system/issues/612)
+
+### Description
+
+Parts of Synergies `<syn-select>`, `<syn-combobox`, `<syn-dropdown>`, `<syn-tooltip>` may get drawn behind other elements, if the `container-type` css property of a parent element is set to something different than `normal`.
+
+### Cause
+
+Older chrome and firefox browser versions as well as current safari browser had a special handling for container queries, which was declared as a "design mistake". For this reason the browsers are already having or getting an updated handling for this and also the specification should be updated in the future.
```

Comment:
I would use `have` instead of `had` because they still show this behavior (at least in Safari)